app/package.json

```
@@ -44,6 +44,7 @@
     "sinon": "^7.3.2"
   },
   "resolutions": {
-    "lodash": ">=4.17.21"
+    "lodash": ">=4.17.21",
+    "ansi-regex": "<6.0.0"
```

shouldn't this rather be 
```suggestion
    "ansi-regex": ">=5.0.1"
```
?

---

They rollbacked it to https://github.com/chalk/ansi-regex/releases/tag/v5.0.1 because v 6.0.1 fails with webpack module errors. However, yarn lock automatically goes for v6.0.1 if you specify >=5.0.1 which breaks it.

---

Should I rather add two entries there? One for >= and the other one for <=? I did not want to specify exact match : "5.0.1."